src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -218,10 +218,8 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
     currency = nativeOnChain(connectedChainId)
   }
 
-  const tokenName = isWrappedNativeToken && currency ? currency.name : tokenDetailData.name
-  const defaultTokenSymbol = tokenDetailData.tokens?.[0]?.symbol ?? token.symbol
-  const tokenSymbol = isWrappedNativeToken && currency ? currency.symbol : defaultTokenSymbol
-
+  const tokenName = tokenDetailData.name
+  const tokenSymbol = tokenDetailData.tokens?.[0]?.symbol?.toUpperCase() ?? token.symbol
```

Comment:
i think vig has mentioned uppercasing in CSS before - would vote to be consistent for whatever we choose